azurerm/resource_arm_packet_capture_test.go

```diff
@@ -0,0 +1,344 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func testAccAzureRMPacketCapture_localDisk(t *testing.T) {
+	resourceName := "azurerm_packet_capture.test"
+
+	ri := acctest.RandInt()
+	location := testLocation()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMPacketCaptureDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAzureRMPacketCapture_localDiskConfig(ri, location),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Minor nitpick: should we be ensuring any attributes are correctly in state? e.g. 

```
resource.TestCheckResourceAttr(resourceName, "storage_location.#", "1"),
resource.TestCheckResourceAttr(resourceName, "storage_location.0.file_path", "/var/captures/packet.cap"),
```

Reply:
so we’ve been leaning on the Import tests to achieve that, since it’s comparing the config anyway - rather than duplicating these values; but I can change this if you want?